client/client.go

```diff
@@ -31,20 +32,29 @@ func NewClient() (tls_client.HttpClient, error) {
 			options = append(options, proxyOption)
 		}
 	} else {
-		_, err := os.Stat("proxy.txt")
-		if err == nil {
-			proxyConfig, readErr := os.ReadFile("proxy.txt")
-			if readErr != nil {
-				fmt.Fprintln(os.Stderr, "Error reading file proxy.txt:", readErr)
-				return nil, readErr
-			}
+		proxyConfigLocations := []string{
+			"proxy.txt",
+			filepath.Join(os.Getenv("HOME"), ".config", "tgpt", "proxy.txt"),
```

Comment:
There is no HOME environment variable on Windows. So use `os.UserHomeDir()`

Reply:
my bad, I originally thought windows will use HOME env like unix, I have made modifications based on your feedback